quasar-core/src/main/java/co/paralleluniverse/fibers/Fiber.java

```diff
@@ -2115,6 +2117,11 @@ public static ByteArraySerializer getFiberSerializer(boolean includeThreadLocals
 
     private static class FiberSerializer extends Serializer<Fiber<?>> {
         private final boolean includeThreadLocals;
+        private final Map<Class<? extends Fiber>, FieldSerializer<Fiber<?>>> fieldSerializers = new HashMap<>();
+
+        private FieldSerializer<Fiber<?>> getFieldSerializer(Kryo kryo, Class<? extends Fiber> fiberClass) {
+            return fieldSerializers.computeIfAbsent(fiberClass, (fc) -> new FieldSerializer<>(kryo, fc));
```

Comment:
Can you drop those brackets around `(fc)` in the lambda? Also, I suspect that parameter type should be
```java
Class<? extends Fiber<?>>
```